FreeRTOS/FreeRTOS-Kernel
# Update MPU wrapper for xTimerGenericCommand API

Description
-----------
This PR updates the MPU wrapper for xTimerGenericCommand API, since in latest FreeRTOS+Kernel code the xTimerGenericCommand API is split into two sub functions

Test Steps
-----------
Build and run Kernel Demo soak tests

Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
